tests/dryrun/join_test.go

```diff
@@ -131,3 +131,28 @@ func TestJoinTCPConnectionsRequired(t *testing.T) {
 
 	t.Logf("%s: test complete", time.Now().Format(time.RFC3339))
 }
+
+func TestJoinRunPreflights(t *testing.T) {
```

Comment:
Is there value in adding this test without asserting anything?

Reply:
Yes, the `dryrun` framework asserts things under the hood already - https://github.com/replicatedhq/embedded-cluster/blob/aeb9ff1f361ca8676e706564aa62d466e12da5be/tests/dryrun/util.go#L48-L55.

As an example all of these errors:
- https://github.com/replicatedhq/embedded-cluster/pull/1567
- https://github.com/replicatedhq/embedded-cluster/pull/1568
- https://github.com/replicatedhq/embedded-cluster/pull/1569

Would have been caught by it.